### PR TITLE
[A11y] Handbook updates for accessibility

### PR DESCRIPTION
**Pull request by heyawhite**
It is imperative that our guidance for content creators is both technically accurate and inclusive. With that, I've made some updates to accessibility guidance. There will be more updates in the future, but this is a good place to start.

Changes proposed in this pull request:

- Updates to the handbook's accessibility guidance
   - Clarify inclusion vs. accessibility needs
   - Add GDDSG references
   - Update guidance for alt text
   - Add guidance on preferred words (and words to avoid)
   - Add plain language guidance, which replaces grade level readability
- Our work should match the guidelines
   - "Keep it simple" to "Keep it concise" -- the jump link still says "simple" to avoid breaking references. This can be updated at a future date.
- Update example code for `<figure>` element which was incorrectly referencing `data=size`

### CODE REVIEW

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:17]:
```suggestion
*  [Writing accessible documentation](https://developers.google.com/style/accessibility)
```

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:25]: Love this addition!

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:64]: Can you add a suggestion to avoid idioms as well?

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:141]: Another excellent addition! I often call this out when editing, it's nice to have a reference to share in feedback to back up my reasoning.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:163]:
```suggestion
*  Mark code blocks with the appropriate formatting (CSS, JSON, and more) to
```

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:171]: Nitpick: This reads weird to me, "Writing about people" sounds much more natural. I know this phrasing is consistent with the rest of the subheadings, but still.

Comment (mihajlija) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:171]: *Saying this as a non-native speaker, so I defer to your judgment.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:192]: Same as above.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:64]: I see now that this is covered under inclusivity. I think it belongs in either place. Happy to keep it under inclusivity because it gets more attention that way.

**Comment by mihajlija** [comment]
Great work Alexandra! One suggestion before merging: double check that none of topics are covered elsewhere in the handbook.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (rachelandrew) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:40]: This heading only makes sense to a US audience (I had to search to work out what grade 8 mapped to in the UK), and might be confusing to an international one as different countries that do have "grades" may use a completely different system. While the links below are helpful, I think a more general definition might be more inclusive as the initial heading. I've seen "School age 13/14" used in various places, granted it doesn't make for a snappy heading. In the UK we would describe that as a "Year 9 reading level".

I think the aim is to "Use straightforward English", which could then be clarified below with that being generally considered as a US Grade 8, school age 13 to 14 level of English.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (rachelandrew) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:171]: I think I prefer "Writing about people", the change sounds like a command "Write about people!" :D

**Review by rachelandrew — COMMENTED** [review]
This all looks good, I put a comment below about the US-centric use of school grades.

The UK Plain English campaign has some really good resources http://www.plainenglish.co.uk/free-guides.html obviously British English but I think they have some good suggestions in their material.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:40]: I agree with the sentiment, but for purposes of international inclusivity, I suggest we also avoid references to British educational system, as well as specific ages. For example in Serbia, they had us reading Tolstoy and the The Epic of Gilgamesh at the age of 14.

"Use straightforward English" is the general message, but the section specifically talks only about using tools to self-edit your writing, so I suggest we change the subheading to be specific about that.

Here's another tool you can add to the list, that I use most of the time: https://hemingwayapp.com/

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (mihajlija) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:171]: Yes, same sentiment here. To be fair, using active voice in titles and subheadings is exactly what we do recommend in the styleguide, but it's more relevant for things like "Use Squoosh to compress images" than this.

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (rachelandrew) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:40]: I wasn't suggesting we referred to the British system, the age thing is used by a number of international sources as comparable to the US-centric grade 8 idea (which appears to be in fairly standard use in the states as a shorthand for "straightforward English").

Comment (mihajlija) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:40]: I understood that, I was just saying that educational systems vary by country, as does the reading proficiency at certain ages. English is the official language in more countries than US and the UK and I don't think it's worthwhile thinking about how do various education systems compare and if it's fair to use any of them as a benchmark for "straightforward English".

All the tooling I came across uses the "grading" system based on US standards, there's no phrasing that can work around that, we can just acknowledge that. 

My proposal is to edit the subheading to be about "using tools to edit your writing to be more readable".

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (heyawhite) [reply src/site/content/en/handbook/inclusion-and-accessibility/index.md:64]: I agree it works in either place, and I hope it gets well-worthy attention in inclusivity. And that folks read this doc, at all!

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (heyawhite) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:171]:
```suggestion
## Writing about people
```

File: src/site/content/en/handbook/inclusion-and-accessibility/index.md
Comment (heyawhite) [inline src/site/content/en/handbook/inclusion-and-accessibility/index.md:192]:
```suggestion
## Writing for an international audience
```

**Comment by heyawhite** [comment]
Any remaining concerns? Otherwise, I think this can be merged.

**Review by mihajlija — APPROVED** [review]

**Pull request merged**
